README.md

```diff
@@ -74,6 +74,31 @@ ant
 ## Environment Variables
 Make a copy of `.env.sample` to `.env` in the env directory.
 
+
+## Docker
+
+This project is dockerized using the LEMP stack. The stack consists of:
+- PHP 7
+- nginx
+
+Inspired by [yii2-dockerized](https://github.com/codemix/yii2-dockerized/blob/master/Dockerfile)
+
+### Installation
+[Mac](https://www.docker.com/docker-mac)
+[Windows](https://www.docker.com/docker-windows)
+[Ubuntu](https://www.docker.com/docker-ubuntu)
+
+### Running Containers
+- Ensure docker is running
+- Change directory to project directory
+- Run the command `docker-compoose up`
```

Comment:
Yeah @tegaphilip. I handled it in this [PR](https://github.com/CottaCush/yii2-base-project/pull/9)